modules/cnf-image-based-upgrade-prep-resources.adoc

```diff
@@ -0,0 +1,431 @@
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  labels:
+    velero.io/storage-location: default
+  name: backup-example-app-lvms
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+  - test
+  includedNamespaceScopedResources:
+  - secrets
+  - persistentvolumeclaims
+  - deployments
+  - statefulsets
+  includedClusterScopedResources:
+  - persistentVolumes
+  - volumesnapshotcontents
+  - logicalvolumes.topolvm.io
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-lvmcluster
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "2" <1>
+spec:
+  backupName:
+    backup-example-lvmcluster
+---
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: restore-example-app-lvms
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+  annotations:
+    lca.openshift.io/apply-wave: "3" <3>
+spec:
+  backupName:
+    backup-example-app-lvms
+  restorePVs: true
+  restoreStatus:
+    includedResources:
+      - logicalvolumes
+----
+<1> Ensure that the `lca.openshift.io/apply-wave` annotation is numerically lower in your {lvms} `Backup` and `Restore` CRs than that of the application so that they are restored before your applications.
+
+[NOTE]
+====
+If you do not define the `lca.openshift.io/apply-wave` annotation in the `Backup` or `Restore` CRs, they will be applied together.
+====
+--
+
+. Generate a `ConfigMap` object for your OADP CRs.
+
+.. Create the `ConfigMap` object:
++
+[source,terminal]
+----
+$ oc create configmap example-oadp-cm --from-file=example-oadp-resources.yaml=<path_to_oadp_crs> -n openshift-adp
+----
+
+. Patch the `ImageBasedUpgrade` CR:
++
+[source,terminal]
+----
+$ oc patch imagebasedupgrades.lca.openshift.io upgrade \
+-p='{"spec": {"oadpContent": [{"name": "oadp-cm-example", "namespace": "openshift-adp"}]}}' \
+--type=merge -n openshift-lifecycle-agent
+----
+
+[id="cnf-image-based-upgrade-creating-backup-extra-manifests_{context}"]
+== (Optional) Creating ConfigMap objects of extra manifests for the image-based upgrade with {lcao}
+
+Create your additional manifests that you want to apply to the target cluster.
+
+.Procedure
+
+. Create a YAML file that contains your extra manifests.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: "pci-sriov-net-e5l"
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames: [ens1f0]
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  mtu: 1500
+  numVfs: 8
+  priority: 99
+  resourceName: pci_sriov_net_e5l
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: "networking-e5l"
+  namespace: openshift-sriov-network-operator
+spec:
+  ipam: |-
+    {
+    }
+  linkState: auto
+  networkNamespace: flexran-testcases
```

Comment:
```suggestion
  networkNamespace: sriov-namespace
```
suggested another namespace to remove specific indications of other well-known solutions